neurom/features/morphology.py

```diff
@@ -589,14 +589,10 @@ def volume_density(morph, neurite_type=NeuriteType.all):
     .. note:: Returns `np.nan` if the convex hull computation fails or there are not points
               available due to neurite type filtering.
     """
-
-    def get_points(neurite):
-        return neurite.points[:, COLS.XYZ]
-
     # note: duplicate points are present but do not affect convex hull calculation
     points = [
         point
-        for point_list in iter_neurites(morph, mapfun=get_points, filt=is_type(neurite_type))
+        for point_list in iter_neurites(morph, mapfun=sf.section_points, filt=is_type(neurite_type))
         for point in point_list
     ]
```

Comment:
I think this could be:
```python
points = [
    point_list.tolist()
    for point_list in iter_neurites(morph, mapfun=sf.section_points, filt=is_type(neurite_type))
]
```
Isn't it better?

Reply:
It wouldn't be a flat list of single points without the second loop.

Reply:
Ah yeah sorry you want a flat list here.